src/program.js

```diff
@@ -103,7 +103,9 @@ export class Program {
       await runCommand(argv);
     } catch (error) {
       const prefix = cmd ? `${cmd}: ` : '';
-      log.error(`\n${prefix}${error.stack}\n`);
+      if (!(error instanceof UsageError) || argv.verbose) {
+        log.error(`\n${prefix}${error.stack}\n`);
+      }
```

Comment:
it may be helpful to use an intermediate variable (like `errorMessage`) so that we only add the `prefix` once.